Setup eslint (#91)

feekosta opened the pull request:
Close #91

feekosta committed:
issue #91
 - setup eslint

miltonbsn commented on package.json:
```diff
@@ -33,5 +33,22 @@
+    "eslint-plugin-react": "^7.20.0",
+    "eslint-plugin-react-hooks": "^2.5.0",
+    "prettier": "^2.0.5",
+    "typescript": "^3.9.3"
```
Is it necessary the typescript dependency here?

feekosta replied:
it was necessary for an airbnb dependency, but I changed it to airbnb-base, and it is no longer needed

miltonbsn commented on .eslintrc:
```diff
@@ -0,0 +1,19 @@
+		"class-methods-use-this": 0,
+		"react/prefer-stateless-function": 0,
+		"react/no-did-update-set-state": 0,
+		"import/no-unresolved": 0,
```
Are these rules necessary since we don't have react around here?

feekosta replied:
I changed the airbnb dependency to airbnb-base and removed all dependencies related to react

feekosta committed:
Merge branch 'master' into setup-eslint

feekosta committed:
issue #91
 - replace airbnb to airbnb-base eslint rules to remove unnecessary dependencies

feekosta committed:
issue #91
 - remove package-lock.json to avoid conflicts

feekosta committed:
issue #91
 - add package-lock.json to .gitignore

idmarjr committed:
Merge branch 'master' into setup-eslint
